source/fab/steps/compile_fortran.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, compiler: str = None, common_flags: List[str] = None, path_fl
         compiler = compiler or os.getenv('FC', 'gfortran -c')
         common_flags = common_flags or []
         env_flags = os.getenv('FFLAGS', '').split()
-        self.exe: str = compiler or os.getenv('FC', 'gfortran -c')  # type: ignore
+        self.compiler: str = compiler or os.getenv('FC', 'gfortran -c')  # type: ignore
```

Comment:
We seem to have two instances of `compiler or os.getenv('FC', 'gfortran -c')`, here and line 65. Is that intended?

Reply:
It's my use of the magic wand merge button again. I _did_ say I would be more vigilant with it's use...sorry!